frontend/packages/topology/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -441,6 +441,13 @@ export const topologyPage = {
       .find('.pf-topology__node.pf-m-warning')
       .contains(nodeName);
   },
+  verifyListNodeAlert: (nodeName: string) => {
+    cy.get('li.pf-c-data-list__item')
+      .contains(nodeName)
+      .parent()
+      .find('div.odc-topology-list-view__alert-cell')
+      .contains('Alerts:');
+  },
```

Comment:
There can be an improvement here:
```suggestion
    cy.get('.odc-topology-list-view__label-cell')
      .contains(nodeName)
      .parent()
      .find('div.odc-topology-list-view__alert-cell')
      .contains('Alerts:');
  },
```
But can be taken as a separate pr

Reply:
Done @sanketpathak